Tests/mock_server.py

```diff
@@ -274,12 +274,13 @@ def start(self, playbook_id, path=None, record=False):
         # Make sure process is up and running
         while not log_file_exists and seconds_since_init < PROXY_PROCESS_INIT_TIMEOUT:
             # Check if log file exist
-            log_file_exists = self.ami.call(['ls', log_file]) == 0
+            log_file_exists = silence_output(self.ami.call, ['ls', log_file], stdout='null', stderr='null') == 0
```

Comment:
what dose this silence_output do? @ShahafBenYakir

Reply:
Making sure the shell commands output won’t appear in the Circle log, only messages/errors we decide to print from this python script will appear.